src/main/java/crazypants/enderio/conduit/AbstractConduit.java

```diff
@@ -639,8 +642,16 @@ protected boolean renderStub(ForgeDirection dir) {
 		  return new GasConduitNetwork();
 	  } else if(IMEConduit.class.isAssignableFrom(type)) {
 		  return new MEConduitNetwork();	
+    } else if (IOCConduit.class.isAssignableFrom(type)) {
+      return new OCConduitNetwork();
```

Comment:
Just reformat the whole thing then, this is more of that code from the broken PR -.-